appendix.Rmd

```diff
@@ -4,6 +4,8 @@
 
 ## dev
 
+* 2019-05-22, update all links eligible to HTTPS and update links to the latest versions of Hadley's books ([`@bisaloo`](https://github.com/bisaloo/), #167)
```

Comment:
maybe write "Hadley Wickham"?

Reply:
oh and now the book is also by Jenny Bryan so I'd suggest writing both names :-)